python/lsst/ts/aos/analysis/nightlyReport.py

```diff
@@ -0,0 +1,591 @@
+                sonic_temp_dev,
+            ) = ([] for _ in range(20))
+            for idx, row in tqdm(unique_day_seq.iterrows(), total=len(unique_day_seq)):
+                day = int(row["day"])
+                seq = int(row["seq"])
+
+                rec_end = row["obs_end"]
+                rec_start = row["obs_start"]
+
+                try:
+                    ringss_data = self.seeing_monitor.getSeeingAtTime(
+                        time=Time(rec_start, scale="utc")
+                    )
+                    ringss_val = ringss_data.fwhmSector
+                except:
+                    ringss_val = np.nan
+
+                # ---------- Environment variables -------------
+                # ----------------------------------------------
+                # Get state
+                try:
+                    total_lut_gravity = [f"lutGravity{i}" for i in range(72)]
+                    total_lut_temperature = [f"lutTemperature{i}" for i in range(72)]
+                    m2_actuator_data = await self.efd_client.select_time_series(
+                        "lsst.sal.MTM2.axialForce",
+                        ["*"],
+                        Time(rec_start, scale="utc"),
+                        Time(rec_start, scale="utc") + TimeDelta(0.1, format="sec"),
+                    )
+
+                    m2_combined_lut = (
+                        m2_actuator_data[total_lut_gravity].values
+                        + m2_actuator_data[total_lut_temperature].values
+                    )
+                    m2_combined_lut = m2_combined_lut.mean(axis=0)
+                    m2_dofs_lut = self.m2_bmf.bending_mode(m2_combined_lut)
+                except:
+                    m2_dofs_lut = np.full(20, np.nan)
+
+                try:
+                    z_cols = [f"zForces{i}" for i in range(156)]
+                    m1m3_el_lut = await self.efd_client.select_time_series(
+                        "lsst.sal.MTM1M3.appliedElevationForces",
+                        z_cols,
+                        Time(rec_start, scale="utc"),
+                        Time(rec_start, scale="utc") + TimeDelta(0.2, format="sec"),
+                    )
+                    m1m3_el_lut = m1m3_el_lut[z_cols].values.mean(axis=0)
+
+                    m1m3_az_lut = await self.efd_client.select_time_series(
+                        "lsst.sal.MTM1M3.appliedAzimuthForces",
+                        z_cols,
+                        Time(rec_start, scale="utc"),
+                        Time(rec_start, scale="utc") + TimeDelta(0.2, format="sec"),
+                    )
+                    m1m3_az_lut = m1m3_az_lut[z_cols].values.mean(axis=0)
+
+                    m1m3_temp_lut = await self.efd_client.select_time_series(
+                        "lsst.sal.MTM1M3.appliedThermalForces",
+                        z_cols,
+                        Time(rec_start, scale="utc"),
+                        Time(rec_start, scale="utc") + TimeDelta(0.2, format="sec"),
```

Comment:
Can you at least tie these three 200ms windows together via a single variable so that if you change this, you can't forget to do them all?